terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# CDF Benchmarking Use Case 1 Environment Setup
+
+![UC 1 Architechture Diagram](./uc1-architechture-diagram.png)
+
+This Terraform script can be used to spin up the GCP resources required for benchmarking the following 
+Use Case 1 pipelines:
+- Pipelines with no transformations (SQLServer -> GCS, SQLServer to BQ)
+- Pipelines with transformations (SQLServer -> Wrangler -> BQ)
+- Pipelines with aggregates (SQLServer -> Wrangler -> GroupBy -> BQ)
+
+Note that the actual pipelines need to be run using the 
+[python scripts in CSR](https://source.developers.google.com/p/quantiphi-datafusion/r/pipeline-automation-scripts) 
```

Comment:
should not add internal source repository here.

Reply:
Removed